pymc3/tests/test_distributions_random.py

```diff
@@ -513,6 +509,22 @@ class TestNormal(BaseTestDistribution):
         "check_pymc_draws_match_reference",
         "check_rv_size",
     ]
+class TestLogitNormal(BaseTestDistribution):
+    def logit_normal_rng_fn(self, rng, size, loc, scale):
+        return expit(st.norm.rvs(loc=loc, scale=scale, size=size, random_state=rng))
```

Comment:
is this the best way to test? I am having a problem here

Reply:
What problem are you having? Maybe it's related to my comment above?